zebra-chain/src/transaction.rs

```diff
@@ -294,6 +294,51 @@ impl Transaction {
         }
     }
 
+    /// Returns the `vpub_old` fields from `JoinSplit`s in this transaction, regardless of version.
+    pub fn sprout_pool(
```

Comment:
Confusing name: There are two different values per JoinSplit (old and new), and two different sprout pools (JoinSplit and chain).

Can you use a name that is specific to `vpub_old` fields from `JoinSplit`s in this transaction?

```suggestion
    /// Returns the `vpub_old` fields from `JoinSplit`s in this transaction, regardless of version.
    ///
    /// This value is removed from the transparent value pool of this transaction, and added to the
    /// sprout value pool.
    pub fn sprout_pool_added_values(
```

Here are the definitions in the spec:
<img width="803" alt="Screen Shot 2021-06-28 at 11 05 41" src="https://user-images.githubusercontent.com/8951843/123565603-d203dc00-d800-11eb-94da-9f86d7a2d52a.png">
<img width="803" alt="Screen Shot 2021-06-28 at 11 07 46" src="https://user-images.githubusercontent.com/8951843/123565701-17280e00-d801-11eb-86c2-884d418bd3da.png">

Reply:
Changed the function name and added the suggested docs at https://github.com/ZcashFoundation/zebra/pull/2399/commits/a204a300f7b02f97a84f69a445f58a93c5921f80